docs/src/doc/user-guide/exporting.md

```diff
@@ -46,16 +46,31 @@ In order to build for Android, set the `isAndroidExportEnabled` flag to `true` i
 On android we do not embed a JVM, we use the existing ART provided by the OS. In order for your game to load the necessary jar files, they need to be converted into dex format. Our gradle plugin will handle this for you, but you need to fulfill the following requirements:
 
 - Android SDK installed
-- `dx` tool resolvable by doing one of the following:
-    - set the `PATH` environment variable to include `<android-sdk-dir>/build-tools/<version>/`
-    - set `dxToolPath` to the file path of `dx`:
+- `d8` tool resolvable by doing one of the following:
+    - (discuraged) only set the `ANDROID_SDK_ROOT` environment variable
```

Comment:
typo ?
Also, we only have 2 methods and the one not discouraged is basically hard writing the path in the Gradle build file,  not really portable in a team using different systems.

Reply:
Well it is portable. You just have to setup your gradle script properly.
You can say in your team that for android to be built, one has to have the `ANDROID_SDK_ROOT` environment variable set and then it would look like this:
```kotlin
d8ToolPath.set("${System.getenv("ANDROID_SDK_ROOT")}/build-tools/30.0.3/d8")
```
Which is very portable IMO.

And what typo are you referring to?

Reply:
discuraged

Reply:
Fixed the typo.
Are you ok with my explanation why i think it's portable or do you disagree?

Reply:
I understand that the "only SDK path" solution is not the most reliable one because the location can change depending on the SDK version. 
But then just write the example you just gave me that combines the SDK path and relative path instead of the "/home/user". The current version gives the feeling you shouldn't use the SDK path at all.

Reply:
Agree. Will change the wording